packages/devtools/lib/src/settings/settings_screen.dart

```diff
@@ -42,31 +50,43 @@ class SettingsScreen extends Screen {
     });
   }
 
+  void _rebuild() {
+    _screenDiv
+      ..clear()
+      ..add(_container..clear());
+    if (serviceManager.hasConnection) {
+      final flutterVersion = div(c: 'flutter-version-container')
+        ..flex(2)
+        ..add(<CoreElement>[
+          CoreElement('h2', text: 'Flutter SDK Version: '),
```

Comment:
nit: there is existing logic elsewhere to determine if you are on Flutter or not. This text should say "Dart SDK Version" if Flutter is not running.

Reply:
Is it correct to use `await serviceManager.connectedApp.isAnyFlutterApp` for this one?

Reply:
that works!